packages/diagram/src/module/diagrams/content/uml/nonNavigableAssociations.ts

```diff
@@ -0,0 +1,35 @@
+import { InterpreterModule, parse } from "@hylimo/core";
+
+/**
+ * Module providing associations without navigation operators
+ */
+export const nonNavigableAssociationsModule = InterpreterModule.create(
+    "uml/nonNavigableConnections",
+    ["common/defaultMarkers"],
+    [],
+    [
+        ...parse(
+            `
+                this.create = scope.internal.createConnectionOperator
+                scope.!-- = create(
+                    startMarkerFactory = scope.defaultMarkers.cross
+                )
+                scope.--! = create(
+                    endMarkerFactory = scope.defaultMarkers.cross
+                )
+                scope.!--! = create(
+                    startMarkerFactory = scope.defaultMarkers.cross,
+                    endMarkerFactory = scope.defaultMarkers.cross
+                )
+                scope.!--> = create(
```

Comment:
From my PSE experience:
I do see the need for non-navigability versions that support aggregations/compositions.
Main question: What would be a reasonable arrow identifier for them?

Reply:
we can do that, however this will be part of a future PR